### ISSUE TITLE
Dark theme variants?

### ISSUE THREAD

**Issue by hjri** [NitruxSA/nitrux-icons#77]
Will there be light icons version that works with dark themes?

**Comment by UriHerrera** [comment]
http://deviantn7k1.deviantart.com/art/Basic-bundle-Nitrux-421219737

**Comment by hjri** [comment]
so dark icons are free from ppa but light ones are paid?

**Comment by UriHerrera** [comment]
Yes. It's always been like this for these and the other icon themes Nitrux has with the exception of Flattr.
